set.mm

```diff
@@ -776576,6 +776859,676 @@ separated by (open) neighborhoods.  See ~ sepnsepo for the equivalence
+
+      ${
+        $d B u v $.  $d C u v w $.  $d ph u v w $.
+        catprs2.l $e |- ( ph -> .<_ = ( le ` C ) ) $.
+        $( A category equipped with the induced preorder is a preordered set,
+           or a proset.  See ~ catprs for a more primitive version.  See
+           ~ prsthinc for constructing a thin category from a proset.
+           (Contributed by Zhi Wang, 18-Sep-2024.) $)
+        catprs2 $p |- ( ph -> C e. Proset ) $=
+          ( vw vv vu cproset cv wbr wa wral wcel catprs ralrimivvva ccat isprsd
+          wi mpbird ) AEPUAMQZUHGRUHNQZGRUIOQZGRSUHUJGRUFSZODTNDTMDTAUKMNODDDAB
+          CDEFGUHUIUJHIJKUBUCAMNODEGUDILKUEUG $.
+      $}
+
+$(
+      catprs3.k @e |- ( ph -> K = ( C sSet <. ( le ` ndx ) , .<_ >. ) ) @.
+      @( A category equipped with the induced preorder is a preordered set, or
+         a proset.  (Contributed by Zhi Wang, XX-Sep-2024.) @)
+      catprs3 @p |- ( ph -> K e. Proset ) @=
+        (  ) ? @.
+$)
+    $}
+
+$(
+    catprs4.f @e |- ( ph -> F = ( c e. Cat |-> .<_ ) ) @.
+    @( There is a function from ` Cat ` to the class of preorders.
+       (Contributed by Zhi Wang, XX-Sep-2024.) @)
+    catprs4 @p |- ( ph -> F : Cat --> ( le " Proset ) ) @=
+      (  ) ? @.
+$)
+  $}
+
+  ${
+    $d B w x y $.  $d H x y $.  $d ph w z $.  $d x y z $.
+    catprsc.1 $e |- ( ph -> .<_ = { <. x , y >. |
+            ( x e. B /\ y e. B /\ ( x H y ) =/= (/) ) } ) $.
+    $( A construction of the preorder induced by a category.  See ~ catprs .
+       (Contributed by Zhi Wang, 18-Sep-2024.) $)
+    catprsc $p |- ( ph -> A. z e. B A. w e. B
+                          ( z .<_ w <-> ( z H w ) =/= (/) ) ) $=
+      ( cv wbr co c0 wne wcel wa w3a vex weq eleq1d wb copab breqd simpl oveq12
+      simpr neeq1d 3anbi123d df-3an bitrdi eqid braba baibd ralrimivva ) ADJZEJ
+      ZHKZUOUPGLZMNZUADEFFAUQUOFOZUPFOZPZUSAUQUOUPBJZFOZCJZFOZVCVEGLZMNZQZBCUBZ
+      KVBUSPZAHVJUOUPIUCVIVKBCUOUPVJDRERBDSZCESZPZVIUTVAUSQVKVNVDUTVFVAVHUSVNVC
+      UOFVLVMUDTVNVEUPFVLVMUFTVNVGURMVCUOVEUPGUEUGUHUTVAUSUIUJVJUKULUJUMUN $.
+  $}
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Examples of categories
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Thin categories
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c ThinCat $.
+
+  $( Extend class notation with the class of thin categories. $)
+  cthinc $a class ThinCat $.
+
+  ${
+    $d B b c f g h x y $.  $d C b c f g h x y $.  $d H b c f g h x y $.
+    $( Definition of the class of thin categories, or posetal categories, whose
+       hom-sets each contain at most one morphism.  Example 3.26(2) of [Adamek]
+       p. 33.  "ThinCat" was taken instead of "PosCat" because the latter might
+       mean the category of posets.  (Contributed by Zhi Wang, 17-Sep-2024.) $)
```

Comment:
The category of posets would typically be denoted simply by **Poset**. But I agree "thin" and `ThinCat` are better anyway.

Reply:
But the syntax token for all categories in `set.mm` ends in **Cat**, like `SetCat`, `RingCat`, etc. ?

Reply:
> But the syntax token for all categories in `set.mm` ends in **Cat**, like `SetCat`, `RingCat`, etc. ?

Oh, I didn't remember, sorry. Anyway, `ThinCat` it is !